be1-go/message/messagedata/transaction_post.go

```diff
@@ -0,0 +1,53 @@
+package messagedata
+
+//TBC
+type TransactionPost struct {
+	Object      string            `json:"object"`
+	Action      string            `json:"action"`
+	Transaction TransactionStruct `json:"transaction"`
+}
+
+//TBC
+type TransactionStruct struct {
```

Comment:
we know this is a struct ;) 
```suggestion
type Transaction struct {
```